### ISSUE TITLE
Strange behavior with the "stretch" parameter set to true.

### ISSUE THREAD

**Issue by micheg** [replit/kaboom#448]
Strange behavior with the "stretch" parameter set to true.
When using the stretch parameter, the positions do not return correct, not even the centering of an object and the objects are no longer clickable.
With the following code:

```
import kaboom from "https://unpkg.com/kaboom@2000.1.6/dist/kaboom.mjs";
let opt =
{
    width: 320,
    height: 240,
    stretch: true
};
kaboom(opt);

loadSprite("bean", "./img/bean.png");

add([
    pos(0, 0),
    rect(320, 240),
    outline(4),
    area(),
])

let player = add([
	sprite("bean"),
	pos(center()),
	area(),
]);

player.onClick((evt)=>{
    addKaboom(mousePos());
});

onClick(()=>{
    console.log(mousePos());
});
```

The expected behavior would be to click on the "bean" and get the balloon, and if you click on the canvas in the lower corner you have reported in the console:
320,240.
Instead it gives the "real" values of the dimension from the canvas, ex: 600,400.

**Comment by lajbel** [comment]
Yeah, that's because the stretch parameter is not working properly yet. Sorry :(

**Comment by micheg** [comment]
This is a pity. We will wait :-)
Without working strecth and letterboxes (and maybe a virtual dpad plugin) it is a bit difficult to develop for mobile devices.

**Comment by slmjkdbtl** [comment]
Should be fixed in #490, plz reopen of problem persists.
